src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/ClientCredentialTypeTests.cs

```diff
@@ -34,7 +34,7 @@ public static void BasicAuthentication_RoundTrips_Echo()
 
             ChannelFactory<IWcfCustomUserNameService> factory = new ChannelFactory<IWcfCustomUserNameService>(basicHttpBinding, new EndpointAddress(Endpoints.Https_BasicAuth_Address));
             factory.Credentials.UserName.UserName = "test1";
-            factory.Credentials.UserName.Password = "test1pwd";
+            factory.Credentials.UserName.Password = "Mytestpwd1";
```

Comment:
This is because we need to create a local user name/pwd on the server machine for this test to work. This is manual currently. The password needs to satisfy the password requirement, hence the change.

As for the server setup, the manual step will be documented on the onenote. We can automate them in the long run.